### ISSUE TITLE
Always "Cheking..."

### ISSUE THREAD

**Issue by AlejandroC90** [rbrs/ubuntu-twitch-indicator#5]
it never shows the channels live on archlinux-gnome, just shows "checking..." all the time
I cloned the git and made a sudo ./setup.sh.

**Comment by rolandasb** [comment]
Hey, can you show me the output?

**Comment by AlejandroC90** [comment]
Here it is, and the icon takes some time to appear, like 3 minutes.
![screenshot from 2015-05-11 05-46-00](https://cloud.githubusercontent.com/assets/5132907/7563289/2d972e56-f7a1-11e4-96a5-d5a39df35346.png)

**Comment by rolandasb** [comment]
This is because you are using python3.4, but indicator is written for python2.7. I have plans to make it available for both versions, but I don't know when exactly.

If you're willing to implement by yourself, pull requests are always welcome :)

**Comment by AlejandroC90** [comment]
Well... I'm not even a beginner in Python so I can't, but I solved my problem:
I searched for the arch wiki and it says: Any program requiring Python 2 needs to point to /usr/bin/python2, instead of /usr/bin/python, which points to Python 3. 
So, I changed that, you just have to change the line to "#!/usr/bin/python2" and it will use the 2 version (I have both them installed).
and it worked perfectly :)
thank you for the answer, sorry for the bad english.
